### DISCUSSION TITLE
How to erase some text in a pdf please ?

### DISCUSSION THREAD

**Discussion by Titi01000** [Stirling-Tools/Stirling-PDF#500] [Q&A]
Hello everyone,
I'm a newcomer to your wonderful Stirling PDF program.
Tonight I wanted to replace one text with another.
I could easily create the new text, but I couldn't delete the old one.
Please tell me which block to go to and where to find the necessary tool.
Thanks in advance.

**Comment by Frooodle** [comment]
Sadly text removal doesn't currently exist in this tool

**Comment by Titi01000** [comment]
Hello Anthony,
Thank you very much for your reply.

As unfortunate as it may be because it is an important function, two questions:
1/ Do you plan to implement it, yes or no ?
2/ If yes, what place has this request among you other priorities ?

Wether we are speaking of 2 weeks or of 3 years, I’ll either qiuetly wait or look for an alternativ solution for this particular need.
Thx in advance for your reply.

**Reply by Frooodle** [reply]
Probably closer to the 1 year side.. it's a difficult thing to do text editting this app is over half a year old and it's something that we still have not been able to do but have tried...

**Comment by avatar1024** [comment]
As a hack you can use the draw function as an "eraser" making it a large white pen (if your doc is on white background).

**Reply by Titi01000** [reply]
Thanks for your help.
It's a smart hack, but I've for the time being, given my preference to a software that does the job without transforming my documents with a bunch of plasters.
This in no way detracts from Sterling PDF's other qualities or from the merits of its development approach.
